src/q5/INGI1341/summary/INGI1341-summary.tex

```diff
@@ -37,33 +49,24 @@ \section{Terminology}
 \section{Layers}
 
 The different layers are represented by the \tabref{layers}.
-A router only has the 3 layers: Network, Datalink and Physical.
+A router only has the 3 layers: Network, Datalink and Physical. TCP/IP model combine Datalink and Physical layer to Link layer
 \begin{table}[!ht]
-  \centering
-  \begin{tabular}{|c|c|c|p{4cm}|c|c|}
-    \hline
-    \multicolumn{3}{|c|}{Layers} & Protocols & PDU\footnote{Protocol Data Unit} & Id\footnote{I know it is a bit an overgeneralization.}\\
-    \cline{1-3}
-    CNP3 & TCP/IP & OSI & & & \\
-    \hline
-    \multirow{3}{*}{Application} & \multirow{3}{*}{Application}             & Application  & BGP, DHCP, DNS, FTP, HTTP, NFS, NTP, RIP, SMTP, SNMP, Telnet & ADU & \\
-    \cline{3-6}
-                                 &                                          & Presentation & MIME, XDR, SSL & & \\
-    \cline{3-6}
-                                 &                                          & Session      & RTP, TLS & SDU & \\
-    \hline
-    Transport                    & Transport                                & Transport    & UDP, MPTCP, TCP, SCTP & segment & Port\\
-    \hline
-    Network                      & Internet                                 & Network      & ICMP, IPsec, IPv4, IPv6, IPX, MPLS\footnote{MultiProtocol Label Switching operates at a layer that is generally considered to lie between traditional definitions of layer 2 (data link layer) and layer 3 (network layer), and thus is often referred to as a ``layer 2.5'' protocol} & packet & IP\\
-    \hline
-    Datalink                     & \multirow{2}{*}{Link}                    & Datalink     & IEEE 802.3 (Ethernet), IEEE 802.4, IEEE 802.5, ATM, ARP, Frame Relay, HDLC, IS-IS, LAPB, LLC, MAC, OSPF, PPP, SLIP & frame & MAC\\
-    \cline{1-1}
-    \cline{3-6}
-    Physical                     &                                          & Physical     & DSL, IEEE 802.3 (Ethernet), IEEE 802.11 (WiFi), ISDN, Modems & bit & \\
-    \hline
-  \end{tabular}
-  \caption{This table contains all the protocols of \cite{bonaventure2011computer}. See \cite{wiki:osimodel} for more protocols.}
-  \label{tab:layers}
+\centering
+	\begin{tabular}{|c|l|c|c|l|}
+	\hline
+	Layer&Feature&PDU&Id&Protocol \\
+	\hline
+	\multirow{2}{*}{Application}&&ADU&&BGP,DHCP,DNS,\\&&&&HTTP,FTP\\
+	\hline
+	\multirow{2}{*}{Transport}&Connectionless unreliable(CRC)&segment&port&UDP,TCP,STCP\\&connection-oriented reliable (graceful) &&&\\
+	\hline
+	\multirow{2}{*}{Network}&Connectionless unreliable(datagram)&Packet&IP&IPv6,Ipv4,\\&connection-oriented(virtual circuit)&&&OSPF,RIP\\
+	\hline
+	\multirow{2}{*}{Datalink}&Unreliable (Wifi)use window-based protocol &Frame&MAC&Ethernet,SLIP\\&Reliable(Wires)&&&Wifi,PPP\\
+	\hline
+	Physical&Unreliable (add/delete/change bit)&bit&&\\
+	\hline
+	\end{tabular}
```

Comment:
I'm not entirely ok with the idea of removing my table :P Maybe we can include both if you don't have enough place :)